cxx/python/algorithms/basic_py.cc

```diff
@@ -16,6 +22,10 @@ PYBIND11_MODULE(basic, m) {
   m.attr("__name__") = "mspasspy.ccore.algorithms.basic";
   m.doc() = "A submodule for algorithms namespace of ccore with common algorithms";
 
+  py::bind_vector<std::vector<double>>(m, "DoubleVector");
+  py::bind_vector<std::vector<TimeSeries>>(m, "TimeSeriesVector");
+  py::bind_vector<std::vector<Seismogram>>(m, "SeismogramVector");
+
```

Comment:
These bindings of vector seem unnecessary. Let me try removing them.

Reply:
Removed fine as expected.

Reply:
Previous comment was not true. It seems the bundle code is never called in any tests, so it is not clear whether they worked or not.

Reply:
Further testing shows that with or without these "bind_vector" lines are equally problematic. With the lines here, we are redefining these "Vector" types under this `mspasspy.ccore.algorithms.basic` module. But because they are also defined in `mspasspy.ccore.seismic`, it turns out they will be treated as different classes. Basically, `mspasspy.ccore.algorithms.basic.TimeSeriesVector == mspasspy.ccore.seismic.TimeSeriesVector ` will be `false`.

However, if we remove these lines, the input and output type of `BundleGroup` will become `List[mspasspy.ccore.seismic.TimeSeries]` and `List[mspasspy.ccore.seismic.Seismogram]`, which is also different from the "Vector" types...

Reply:
This is fixed by https://github.com/wangyinz/mspass/pull/106/commits/0f4f732c75de07c788357785f3ea538325ecef84